docs/content/docs/connectors/table/datagen.md

```diff
@@ -253,6 +259,13 @@ Connector Options
       <td>(Type of field)</td>
       <td>Maximum value of random generator, work for numeric types.</td>
     </tr>
+    <tr>
+      <td><h5>fields.#.max-past</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">0</td>
+      <td>Duration</td>
+      <td>Maximum past of timestamp random generator， work for timestamp types.</td>
```

Comment:
```suggestion
      <td>Maximum past of timestamp random generator, only work for timestamp types.</td>
```

And please also mention this can be used to generate out-of-order records.